pkg/apiutil/apiutil.go

```diff
@@ -239,3 +249,19 @@ func doJSON(client *http.Client, req *http.Request, checkOpts ...func([]byte, in
 	}
 	return nil
 }
+
+func doJSONIgnoreRespStatus(client *http.Client, req *http.Request, checkOpts ...func([]byte, int)) error {
```

Comment:
It has many same code with do doJSON,  may exract one function doJSON(client *http.Client, req *http.Reques, needOK bool,checkOpts ...func([]byte, int))?

Reply:
> It has many same code with do doJSON, may exract one function doJSON(client *http.Client, req *http.Reques, needOK bool,checkOpts ...func([]byte, int))?

update